app/models/user_upload.rb

```diff
@@ -10,13 +10,17 @@ def make_object(user_data)
                           address: user_data["address"],
                           email: user_data["e-mail"],
                           phone: user_data["phone"],
+                          notes: user_data["notes"],
                           password: "password" )
+      ImportedUserMailer.welcome_email(user).deliver
```

Comment:
Let's not call this here, let's just do it in the console manually because time's too short to make a cron job for it right now.